mage_ai/orchestration/db/models.py

```diff
@@ -128,8 +144,9 @@ def current_execution_date(self) -> datetime:
                 timedelta(days=now.weekday())
         elif self.schedule_interval == '@monthly':
             return now.replace(second=0, microsecond=0, minute=0, hour=0, day=1)
-        # TODO: Support cron syntax
-        return None
+        else:
```

Comment:
can schedule_interval be None?

Reply:
@wangxiaoyou1993 what would happen if schedule_interval is None

Reply:
It shouldn't schedule if schedule_interval is None. We should return None in current_execution_date here if schedule_interval is None.

Reply:
👍